api/src/test/java/org/openmrs/module/locationbasedaccess/aop/LocationUtilsTest.java

```diff
@@ -0,0 +1,120 @@
+package org.openmrs.module.locationbasedaccess.aop;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import org.openmrs.Location;
+import org.openmrs.Person;
+import org.openmrs.PersonAttributeType;
+import org.openmrs.User;
+import org.openmrs.api.LocationService;
+import org.openmrs.api.PersonService;
+import org.openmrs.api.UserService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.locationbasedaccess.LocationBasedAccessConstants;
+import org.openmrs.module.locationbasedaccess.utils.LocationUtils;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class LocationUtilsTest extends BaseModuleContextSensitiveTest {
+
+	private PersonService personService;
+	private LocationService locationService;
+	private UserService userService;
+	private static final String XML_FILENAME_WITH_PERSON_DATA = "include/LocationUtilTestData.xml";
+	private static final int DEMO_PERSON3_ID = 3;
+	private static final int DEMO_PERSON2_ID = 2;
+	private static final int DEMO_PERSON4_ID = 4;
+	private static final int DEMO_PERSON5_ID = 6005;
+	private static final int DEMO_PERSON6_ID = 6006;
+	private static final int DEMO_LOCATION1_ID = 1;
+	private static final int DEMO_LOCATION2_ID = 2;
+	private static final int DEMO_USER1_ID = 6001;
+	private static final int DEMO_USER2_ID = 6002;
+	private static final int DEMO_USER3_ID = 6003;
+
+	public LocationUtilsTest(){
+
+	}
+
+	@Before
+	public void setUp() throws Exception {
+		userService = Context.getUserService();
+		personService = Context.getPersonService();
+		locationService = Context.getLocationService();
+		executeDataSet(XML_FILENAME_WITH_PERSON_DATA);
+	}
+
+	@Test
+	public void getPersonLocation_shouldReturnPersonLocation(){
+		Person person3 = personService.getPerson(DEMO_PERSON3_ID);
+		Location location1 = locationService.getLocation(DEMO_LOCATION1_ID);
+		Location location3 = LocationUtils.getPersonLocation(person3);
+		assertEquals(location3,location1);
+		Person person4 = personService.getPerson(DEMO_PERSON4_ID);
+		Location location2 = locationService.getLocation(DEMO_LOCATION2_ID);
+		Location location4 = LocationUtils.getPersonLocation(person4);
+		assertEquals(location2,location4);
+	}
+
+	@Test
+	public void getPersonLocation_shouldReturnNullIfNoLocationAttribute() {
+		Person person2 = personService.getPerson(DEMO_PERSON2_ID);
+		Location location = LocationUtils.getPersonLocation(person2);
+		assertNull(location);
+	}
+
+	@Test
+	public void doesPersonBelongToGivenLocation_True_ifPersonBelongToSameLocation(){
```

Comment:
Can you change this method name to a meaning full? 
Eg : does<A>ShouldBeTrue_If<B>